scripts/python/setup.py

```diff
@@ -1,10 +1,11 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
I don't want to make that change. If someone still has Python 2, or a Python binary "python" (e.g. my Python 3.6 binary) it will fail.